src/libraries/System.Net.Http/src/System/Net/Http/Headers/CacheControlHeaderValue.cs

```diff
@@ -367,7 +367,7 @@ public static CacheControlHeaderValue Parse(string? input)
             return (CacheControlHeaderValue)CacheControlHeaderParser.Parser.ParseValue(input, null, ref index);
         }
 
-        public static bool TryParse(string? input, out CacheControlHeaderValue? parsedValue)
+        public static bool TryParse([NotNullWhen(true)] string? input, [NotNullWhen(true)] out CacheControlHeaderValue? parsedValue)
```

Comment:
You are right. I think that I miss understood the original problem. If I quote the ticket, the solution is to "Returning empty value instead of null sounds like the best option - least impactful on users."

Is this better ? If you see any improvment, let me now, I'll fix it